common/proto/dex_amm.proto

```diff
@@ -5,23 +5,259 @@ import "erc20.proto";
 
 package messari.dex_amm.v1;
 
-message Pool {
-  string name = 1;
-  string address = 2; // bytes: Address
-  repeated messari.erc20.v1.ERC20Token input_tokens = 3;
+message Token {
```

Comment:
Why new `Token` type? We should building on existing types when possible.

Reply:
yes, I have changed this to use already existing types.